src/vs/server/node/server.cli.ts

```diff
@@ -339,6 +347,10 @@ export async function main(desc: ProductDescription, args: string[]): Promise<vo
 		if (waitMarkerFilePath) {
 			waitForFileDeleted(waitMarkerFilePath);
```

Comment:
@aeschli @alexdima do you maybe recall why `waitForFileDeleted` is not awaited? It seems to return a `Promise` but not await it. I failed to find the history for this change.

Reply:
I don't remember, looks like an oversight

Reply:
Created https://github.com/microsoft/vscode/issues/210034